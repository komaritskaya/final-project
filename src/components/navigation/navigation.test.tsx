import React from 'react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import Navigation from './navigation';

const mockStore = configureStore([]);

it('Should Navigation component render correctly', () => {
  const store = mockStore({
    app: {
      filter: 'default',
    },
  });

  const node = renderer.create(
    <Provider store={store}>
      <Navigation />,
    </Provider>,
  ).toJSON();

  expect(node).toMatchSnapshot();
});
