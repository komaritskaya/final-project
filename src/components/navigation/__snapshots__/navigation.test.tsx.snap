// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Navigation component render correctly 1`] = `
Array [
  <ul>
    <li>
      <div
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        default
      </div>
    </li>
    <li>
      <div
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        caught
      </div>
    </li>
  </ul>,
  ",",
]
`;
